src/function/macro_function.cpp

```diff
@@ -6,12 +6,23 @@
 #include "duckdb/parser/expression/columnref_expression.hpp"
 #include "duckdb/parser/expression/comparison_expression.hpp"
 #include "duckdb/parser/expression/function_expression.hpp"
+#include "duckdb/parser/expression/constant_expression.hpp"
 
 namespace duckdb {
 
-MacroFunction::MacroFunction(unique_ptr<ParsedExpression> expression) : expression(move(expression)) {
+MacroFunction::MacroFunction(unique_ptr<ParsedExpression> expression) : expression(move(expression) ){
 }
 
+MacroFunction::MacroFunction(void){
+ }
+
+ bool MacroFunction::is_query() {
```

Comment:
This does not follow function naming conventions, should be `IsQuery()`.

Reply:
Will refactor the name